.gitignore

```diff
@@ -81,3 +81,14 @@ custom-wallet-config.nix
 # wallet web API benchmarking results
 wallet-new/bench/results/*.csv
 wallet-new/bench/results/*.txt
+
+# cardano-state-* for wallet data
+cardano-state-*
+state-wallet-mainnet
```

Comment:
Please change this to `state-*` (see https://github.com/input-output-hk/cardano-sl/pull/2673/files#r174981381)